site/README.md

```diff
@@ -24,33 +24,62 @@ This directory contains the code for the
 a sub-directory of the
 [Apache Calcite web site](https://calcite.apache.org).
 
-## Setup
+You can build the site manually using your environment or use the docker compose file.
+
+## Manually
+
+### Setup your environment
 
 Similar to the instructions to
-[set up the Calcite web site](../site/README.md).
+[set up the Calcite web site](https://github.com/apache/calcite-avatica/blob/master/site/README.md).
 
-Site generation currently works best with ruby-2.4.1.
+Site generation currently works best with ruby-2.5.1.
 
 1. `cd site`
 2. `svn co https://svn.apache.org/repos/asf/calcite/site/avatica target/avatica`
-3. `sudo apt-get install rubygems ruby2.1-dev zlib1g-dev` (linux)
-4. `sudo gem install bundler github-pages jekyll jekyll-oembed`
```

Comment:
Did removing this line happen on purpose? I think at least `bundler` needs to be there before `bundle install`

Reply:
done.